maven/bnd-export-maven-plugin/pom.xml

```diff
@@ -41,6 +41,10 @@
             <groupId>biz.aQute.bnd</groupId>
             <artifactId>biz.aQute.repository</artifactId>
         </dependency>
+        <dependency>
+            <groupId>biz.aQute.bnd</groupId>
+            <artifactId>bnd-maven-plugin-lib</artifactId>
```

Comment:
Perhaps we can call this `bnd-shared-maven-lib`? This seems to fit the existing naming better. After `bnd-` is the _purpose_ and at the end is the type. This is a _lib_ rather than a _plugin_.

Another idea (which may be stupid given my general maven lack-of-knowledge), would be to put the shared code in the parent project? Crazy?